RavenDB.Identity/UserStore.cs

```diff
@@ -798,12 +798,14 @@ public Task<int> CountCodesAsync(TUser user, CancellationToken cancellationToken
         /// <returns></returns>
         public static void MigrateToV6(IDocumentStore docStore)
         {
+            var collectionName = docStore.Conventions.FindCollectionName(typeof(IdentityUserByUserName));
+
             // Step 1: find all the old IdentityByUserName objects.
             var emails = new List<(string userId, string email)>(1000);
             using (var dbSession = docStore.OpenSession())
             {
 #pragma warning disable CS0618 // Type or member is obsolete
-                var stream = dbSession.Advanced.Stream<IdentityUserByUserName>("IdentityUserByUserNames/");
+                var stream = dbSession.Advanced.Stream<IdentityUserByUserName>("{collectionName}/");
```

Comment:
Did you mean $"{collectionName}/" (notice the dollar sign)?

Reply:
You are right. Fixed.